weave/trace/weave_init.py

```diff
@@ -185,6 +184,8 @@ def init_weave_get_server(
 
 
 def init_local() -> InitializedClient:
+    from weave.trace_server import sqlite_trace_server
```

Comment:
red flag: did we introduce an import loop?

Reply:
not afaik

Reply:
then why move here?

Reply:
This file should not depend on `trace_server` at all since the contents of that dir will move into `core` in future.  This is a step in that direction.

Also, afaik the `init_local` and `init_local_client` are not actually used by end-users atm.  It's just there for internal use against sqlite